app/src/main/res/layout/fragment_home.xml

```diff
@@ -41,34 +40,35 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@id/menuButton" />
 
-    <FrameLayout
-        android:id="@+id/toolbar_wrapper"
-        android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:layout_marginLeft="16dp"
-        android:layout_marginTop="64dp"
-        android:layout_marginRight="16dp"
-        android:background="@drawable/home_search_background"
-        android:clipToPadding="false"
-        android:elevation="5dp"
-        android:outlineProvider="paddedBounds"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@id/wordmark">
+    <org.mozilla.fenix.home.SearchView
+            android:id="@+id/toolbar_wrapper"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="16dp"
```

Comment:
Please use Start and End for bidirectional locales. Left and Right should be eliminated if we're not supporting ancient Android versions.